README.adoc

```diff
@@ -6,6 +6,133 @@ Rocket implementation of the Application Container Specification.
 The goal is to make Rocket available as an installable package for any
 distribution which uses RPM as the binary distribution mechanism.
 
+A further goal is to get the packaging files incorporated into the
+upstream source tree for the Rocket process so that packaging is
+considered a first-class part of the development and build process.
+
+== Package Files
+
+The Rocket package will consist of three (3) binary files and a set of
+supporting files and directories.  The placement of the files will be
+consistent with the
+https://fedoraproject.org/wiki/Packaging:Guidelines[Fedora Packaging
+Guidelines].
+
+```
+/usr/bin/actool
```

Comment:
they do have it in the build script yes, but the actool in there is built from within the Godeps dir, the actual upstream for it would be https://github.com/appc/spec and it's currently active